apps/percona/functional/snapshot_rebuild/run_litmus_test.yml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: litmus-k8s-snapshot-rebuild-
+  namespace: litmus
+spec:
+  template:
+    metadata:
+      name: litmus
+      labels:
+        app: k8s-snapshot-litmus-rebuild
```

Comment:
Sorry to nitpick - may be trivial - I suppose all 'chaos tests ' use the `name: <label>` to differentiate it from app deployers/workloads ( Somehow felt the categorization helps while adding gitlab runners)